lib/src/rules/flutter_style_todos.dart

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2018, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/ast/token.dart';
+import 'package:analyzer/dart/ast/visitor.dart';
+import 'package:linter/src/analyzer.dart';
+
+const _desc = r'Use Flutter TODO format.';
```

Comment:
to make this more useful, maybe `r'Use Flutter TODO format: // TODO(username): message, https://URL-to-issue.';`

Reply:
done